grails-app/services/com/recomdata/transmart/data/export/HighDimExportService.groovy

```diff
@@ -6,6 +6,8 @@ import org.transmartproject.core.dataquery.highdim.AssayColumn
 import org.transmartproject.core.dataquery.highdim.HighDimensionDataTypeResource
 import org.transmartproject.core.dataquery.highdim.assayconstraints.AssayConstraint
 import org.transmartproject.core.dataquery.highdim.projections.Projection
+import org.transmartproject.core.ontology.OntologyTerm
+import org.transmartproject.db.concept.ConceptFullName
```

Comment:
Not really, just that the tests need to be moved too, so it would be better to add a .travis.yml to core-api and add the core-api to travis.